src/lexer.rs

```diff
@@ -512,7 +512,7 @@ where
     }
 
     fn lex_comment(&mut self) -> TokenResult {
-        assert_eq!(self.input.peek_multiple(2), "/*");
+        debug_assert!(self.input.peek_multiple(2) == "/*");
```

Comment:
Use https://doc.rust-lang.org/std/macro.debug_assert_eq.html, for better failure reporting

Reply:
Oh this exists? I searched for it, but couldn't find it. oops :smile: